core/http/workbench/src/main/webapp/scripts/create.js

```diff
@@ -19,8 +19,9 @@ var workbench;
  */
 function checkOverwrite() {
     var submit = false;
+    var id = workbench.create.id.val();
```

Comment:
Please note that our javascript files are compiled from the corresponding typescript (in webapp/scripts/ts). You should not edit the javascript directly, but instead make the change in the typescript file and then recompile the ts to javascript (using the compileTypescript.sh found in the workbench root dir)>